jsk_baxter_robot/jsk_baxter_startup/baxter.launch

```diff
@@ -188,6 +188,9 @@
     <include file="$(find jsk_baxter_startup)/jsk_baxter_lifelog/db_client.launch" >
       <arg name="twitter" value="$(arg launch_twitter)"/>
     </include>
+    <include file="$(find influxdb_store)/sample/baxter_influxdb_logger.launch">
```

Comment:
Please add `influxdb_store` to package.xml.

In addition, `influxdb_store` is not yet released.
https://github.com/jsk-ros-pkg/jsk_3rdparty/tree/master/influxdb_store
I understand you are currently using https://github.com/knorth55/eus_vive/blob/07714b0503c49a5a094eab52671ab40f9e95806e/baxter.rosinstall#L17-L20 and now on the way of migration. Is my understanding correct?

Reply:
yes correct

Reply:
OK, thank you.